.github/workflows/tests.yaml

```diff
@@ -0,0 +1,119 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  Linting:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0
+
+  Test:
+    needs: Linting
+    name: ${{ matrix.os }}, ${{ matrix.env }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        postgis: [false]
+        dev: [false]
+        env:
+          - ci/travis/36-minimal.yaml
+          - ci/travis/38-no-optional-deps.yaml
+          - ci/travis/36-pd025.yaml
+          - ci/travis/37-latest-defaults.yaml
+          - ci/travis/37-latest-conda-forge.yaml
+          - ci/travis/38-latest-conda-forge.yaml
+        include:
+          - env: ci/travis/37-latest-conda-forge.yaml
+            os: macos-latest
+            postgis: false
+            dev: false
+          - env: ci/travis/38-latest-conda-forge.yaml
+            os: macos-latest
+            postgis: false
+            dev: false
+          - env: ci/travis/37-latest-conda-forge.yaml
+            os: windows-latest
+            postgis: false
+            dev: false
+          - env: ci/travis/38-latest-conda-forge.yaml
+            os: windows-latest
+            postgis: false
+            dev: false
+          - env: ci/travis/37-dev.yaml
+            os: ubuntu-latest
+            dev: true
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Conda
+        uses: s-weigand/setup-conda@v1.0.4
```

Comment:
Is there a reason you switched to a different action? 
(the one of goanpeca now moved to an conda repo, so seems like an "official" supported one: https://github.com/conda-incubator/setup-miniconda)

Reply:
Only because of those warnings. This one seemed to do the job without any warnings. I figure we can always switch back whenever those warnings are fixed (or if GitHub fixes the UI for marking as allowed failure).

Reply:
This one also sets the pre-installed conda, which makes it faster, the incubator one downloads latest miniconda3. TL;DR I'd stick with this one for CIs but bare in mind that future issues may be due to an outdated version of conda.